### ISSUE TITLE
cli doesnt run

### ISSUE THREAD

**Issue by idkjs** [lachenmayer/graphql-scraper#1]
```sh
$[master *] npm ls -g graphql-scraper
/Users/prisc_000/.npm-packages/lib
└── graphql-scraper@1.1.0

$[master *] graphql-scraper
bash: graphql-scraper: command not found

```

**Comment by lachenmayer** [comment]
Thanks for catching this - I think you're officially the first user of the CLI ;)
Fixed & published as `v1.2.1`.
Let me know if that works for you, thanks!

**Comment by idkjs** [comment]
Glad to help. And thanks for sharing. Update works!
